tkweb/apps/eval/templates/evalmacros/begin_hide.html

```diff
@@ -1,5 +1,7 @@
-<a class="btn btn-default btn-xs" role="button" data-toggle="collapse" href="#{{ id }}" aria-expanded="{{ expanded }}" aria-controls="{{ id }}">
-    {{ title }}
-</a>
-<div class="collapse {% if expanded %}in{% endif %}" id="{{ id }}">
-    <div class="well">
+<div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="heading{{year}}">
```

Comment:
Hvad betyder `id="heading{{year}}"`? Det ligner for mig en fejl, da jeg vil tro at `{{ year }}` ikke er defineret i denne template, eller i hvert fald er defineret som det samme hver gang `begin_hide` bruges?

Reply:
Du har ret. Det er fikset.